langchain/src/agents/tests/AWSLambda.test.ts

```diff
@@ -0,0 +1,39 @@
+import { test, jest, expect } from "@jest/globals";
+// eslint-disable-next-line import/no-extraneous-dependencies
+import LambdaClient from "@aws-sdk/client-lambda";
+import { AWSLambda } from "../tools/AWSLambda.js";
+
+// eslint-disable-next-line tree-shaking/no-side-effects-in-initialization
+jest.mock("@aws-sdk/client-lambda", () => ({
```

Comment:
For some reason eslint didn't like my use of `jest.mock` here, so I disabled the linter.